actionview/lib/action_view/helpers/form_tag_helper.rb

```diff
@@ -449,7 +449,8 @@ def submit_tag(value = "Save changes", options = {})
       # JavaScript, for example. You can use the button tag as a regular
       # submit tag but it isn't supported in legacy browsers. However,
       # the button tag allows richer labels such as images and emphasis,
```

Comment:
@rafaelfranca - oops.... confusing thumbs up..... do you want me to change it or it is good as it is? :smile: